.github/workflows/triage.yml

```diff
@@ -0,0 +1,26 @@
+name: Triage
+
+on:
+  pull_request_target:
+    types: [opened]
+#   issues:
+#     types: [opened]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull Request Labeler
+#         if: github.event_name == 'pull_request_target'
+        uses: actions/labeler@v3
+        with:
+          configuration-path: .github/pr-labeler-config.yml
```

Comment:
So, this is our local file right ? But we do not have any file like that ?

Reply:
Now we have
https://github.com/pytorch/ignite/pull/1835/files#diff-c07d414af06346d8cbebb019db1e0079729e57365ed5954a77058bca6c53bb62

Reply:
after discussion https://github.com/pytorch/ignite/pull/1835#discussion_r598448898
we decided to rename it

Reply:
OK, I see, my bad